GOVERNANCE.md

```diff
@@ -0,0 +1,68 @@
+# Project Governance
+
+This document outlines the governance model for Weld projects. This governance model is designed to uphold the principles of transparency, open collaboration, and community involvement.
+
+## Roles and Responsibilities
+
+**Contributors**: Anyone who contributes to Weld projects in any form.
+
+**Maintainers**: Contributors eligible for write access to a Weld projects code repository.
+Responsible for driving initiatives and reviewing/merging contributions in their area of expertise.
+
+**Project Lead**: Maintainer (at most one per code repository) with higher decision power.
+Responsible for keeping Weld projects' technical direction consistent, safe and sustainable. This involves in particular enforcing compliance with requirements of the Commonhaus Foundation.
+
+## Decision-Making
+
+### Consensus-seeking (lazy consensus)
+
+Weld projects primarily aims for a consensus-based decision-making process, where Maintainers and active Contributors discuss and come to an agreement.
+In practice, this involves:
+Discussing matters openly, to facilitate others joining the discussions and expressing concerns.
+Taking into account every Contributor’s opinion, regardless of their role.
+
+Actual implementation of consensus decision-making is up to Maintainers and can vary based on the audience and criticality of the discussion. Inspiration may be found in the [Lazy Consensus model as defined by the Apache Foundation](https://community.apache.org/committers/decisionMaking.html), and in [Martha’s Rules](https://digitalcommons.unl.edu/cgi/viewcontent.cgi?article=1825&context=sociologyfacpub).
+
+### Conflict Resolution
+
+If conflicts arise, Maintainers are responsible for facilitating a resolution.
+
+The Project Lead holds the power to make the final decision.
+
+As a last resort, in particular in case of disagreement about the decision-making process, the Commonhaus Foundation Council (CFC) can be asked to mediate the discussion.
+
+## Role granting/revoking
+
+The role of Maintainer or Project Lead or Overall Project Lead is granted or revoked through the decision-making process, with additional restrictions:
```

Comment:
Seems like the definition of the `Overall Project Lead` is missing from the `Roles and Responsibilities` section 🙂

Reply:
Pending in https://github.com/weld/.github/pull/3